clusterctl/examples/vsphere/machines.yaml.template

```diff
@@ -0,0 +1,62 @@
+items:
+- apiVersion: "cluster.k8s.io/v1alpha1"
+  kind: Machine
+  metadata:
+    name: tf-test-master
+    labels:
+      set: master
+  spec:
+    providerConfig:
+      value:
+        apiVersion: "terraformproviderconfig/v1alpha1"
+        kind: "TerraformProviderConfig"
+        terraformMachine: "standard-master"
+        terraformVariables: [
+         	"user = \"\"",
+          "password = \"\"",
+          "vsphere_server = \"",
+          "datacenter = \"\"",
+          "datastore = \"\"",
+          "resource_pool = \"\"",
+          "network = \"\"",
+          "num_cpus = \"2\"",
+          "memory = \"2048\"",
+          "vm_template = \"\"",
+          "disk_label = \"\"",
+          "disk_size = \"\"",
+          "virtual_machine_domain = \"\"",
+				]
+    versions:
+      kubelet: 1.10.1
+      controlPlane: 1.10.1
+    roles:
+    - Master
+- apiVersion: "cluster.k8s.io/v1alpha1"
+  kind: Machine
+  metadata:
+    generateName: jesschen-tf-test-
+  spec:
+    providerConfig:
+      value:
+        apiVersion: "terraformproviderconfig/v1alpha1"
+        kind: "TerraformProviderConfig"
+        terraformMachine: "standard-node"
+        terraformVariables: [
+         	"user = \"\"",
+          "password = \"\"",
+          "vsphere_server = \"",
+          "datacenter = \"\"",
+          "datastore = \"\"",
+          "resource_pool = \"\"",
+          "network = \"\"",
+          "num_cpus = \"2\"",
+          "memory = \"2048\"",
+          "vm_template = \"\"",
+          "disk_label = \"\"",
+          "disk_size = \"\"",
+          "virtual_machine_domain = \"\"",
+				]
+    versions:
+      kubelet: 1.9.7
```

Comment:
is it intentional to have the master run at 1.10.1 while the nodes run at 1.9.7
if so, a comment would be nice.

Reply:
Fixed.